From 533b46ef357c95ee33c69897ec15458fd081bcb8 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 10 Jul 2019 18:05:57 -0700
Subject: [PATCH] TRITON-580 urclient could include some more details about
 who's running the command TOOLS-1308 node-urclient should support automatic
 reconnection TOOLS-1332 ur client does not close HTTP server on close()
 TOOLS-1357 urclient should terminate inflight requests on AMQP abend
 TOOLS-1009 update libuuid to 0.1.4 (urclient)

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 6956625..74c2065 100644
--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
         "tab": "0.1.0",
         "triton-netconfig": "1.4.0",
         "ufds": "1.6.0",
-        "urclient": "1.0.0",
+        "urclient": "1.2.1",
         "vasync": "1.6.3",
         "verror": "1.10.0",
         "wordwrap": "1.0.0",
-- 
2.21.0

